source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -77,7 +77,7 @@ If your PSP is Stripe, you can sign in to the [GOV.UK Pay admin tool](https://se
 - payments ('payouts') that Stripe has made to your bank account
 - which of your users' payments are in each payout
 
-From 6 April 2022, [if Stripe charges fees on a failed payment](/reporting/#net-amount), they will deduct the fees from your Stripe balance. Stripe bundles the failed payments into your regular payouts. Until 6 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments.
+From 6 April 2022, [if Stripe charges fees on a failed payment](/reporting/#psp-fees), they will deduct the fees from your Stripe balance. Stripe bundles the failed payments into your regular payouts. Until 6 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments.
```

Comment:
```suggestion
From 6 April 2022, if Stripe charges fees on a failed payment, they will deduct the fees from your Stripe balance. You can [read more about PSP fees](/reporting/#psp-fees). Stripe bundles the failed payments into your regular payouts. Until 6 April 2022, GOV.UK Pay will cover the cost of transaction fees on failed payments.
```

Reply:
You say this at a different stage ... could just use this here? "Stripe may charge transaction fees on failed payments. If this happens, `net_amount` will be a negative number. From 6 April 2022, Stripe will take negative net amounts from your Stripe account balance. For example, a `net_amount` of `-200` on a payment will take £2.00 from your Stripe account."

Reply:
This page is more of a overview of integrating with the API, rather than getting into the nitty gritty of API elements like `net_amount`. I do think the wording here should be closer to your suggestion though, so I'll make a few changes.